test/runtime/samples/element-source-location/_config.js

```diff
@@ -11,14 +11,14 @@ export default {
 
 		assert.deepEqual(h1.__svelte_meta.loc, {
 			file: path.relative(process.cwd(), path.resolve(__dirname, 'main.svelte')),
-			line: 4,
+			line: 5,
```

Comment:
I've reverted these changes since it seems like there's some other bug that was present previously. Perhaps something like https://github.com/Rich-Harris/magic-string/issues/248 which was causing some of our other source map tests to assert unexpected behavior